### DISCUSSION TITLE
.Bib file not found for bibliography in qmd

### DISCUSSION THREAD

**Discussion by briancluyse** [quarto-dev/quarto-cli#3932] [Q&A] [answered]
Hello everyone

I am fairly new to Quarto and I have the following problem:
I am using a .bib file (generated with Better Bib Text to easy with citations) as a source file (bibiliography) for my Quarto document. The bib file, as well as the csl file I use, are in the same folder as my RProj. and other related files as this is required to use the 'bibliography:' function in the YAML metadata. However, when add the bib file (called references.bib) to the bibliography function like this

bibliography: references.bib

I get an error that the file cannot be found, although it is in the same directory. When I add the path itself "C:/Users/RStudio/references.bib" R cannot find it either.

What should I do?

**Comment by dragonstyle** [comment, marked as answer]
Would it be possible to share the source code that is causing the trouble? It should be as a simple as a relative path to the bib file.

**Reply by briancluyse** [reply]
I tried it again and this time, without having changed anything, really, it suddenly works. Strange... Nonetheless, thank you for wanting to help!

**Reply by dragonstyle** [reply]
Sure thing, glad its working.
